### ISSUE TITLE
Naming for the library

### ISSUE THREAD

**Issue by ilblackdragon** [near/chainsig-lib#2]
Generally we have a pattern of near-{*}-js for libraries used in frontend/clients. Should we follow the same standard for this library?

**Comment by frol** [comment]
@mattlockyer Indeed, there is an established pattern of naming that include the target language, so we avoid confusion:

* borsh-js / borsh-rs
* near-api-js / near-jsonrpc-client-rs
* near-sdk-js / near-sdk-rs

Please, consider that and also including `near-` prefix if applicable, so when contributors fork the project under their account name, it does not collide with anything else (e.g. github.com/near/docs is pretty unfortunate name since after forking it it becomes github.com/frol/docs)
